packages/@okta/vuepress-site/docs/guides/add-an-external-idp/before-you-begin/index.md

```diff
@@ -19,6 +19,7 @@ This guide assumes that you:
 
 We support a lot of Identity Providers. This guide provides instructions for the following Identity Providers. If the provider that you need isn't listed, we may still support it through generic OpenID Connect or SAML. The Identity Provider's documentation should say which protocol you need to use.
```

Comment:
Space before "We.."

Reply:
Hmmm no space in the file...that's weird. looks fine when I build locally....
<img width="489" alt="Screen Shot 2020-01-16 at 9 29 39 AM" src="https://user-images.githubusercontent.com/45509385/72543388-de7dba00-3842-11ea-99ae-0df8f5e06799.png">